deploy/helm/values.yaml

```diff
@@ -0,0 +1,101 @@
+# Default values for the starboard-operator Helm chart, these are used to render
+# the templates into valid k8s Resources.
+
+# targetNamespace defines where you want starboard-operator to operate. By
+# default it will only operate in the namespace its installed in, but you can
+# specify another namespace, or a comma separated list of namespaces, or set it
+# to a blank string to let it operate in all namespaces.
+targetNamespaces: "{{ .Release.Namespace }}"
+
+nameOverride: ""
+fullnameOverride: ""
+
+image:
+  repository: "aquasec/starboard-operator"
+  # tag is an override of the image tag, which is by default set by the
+  # appVersion field in Chart.yaml.
+  tag: ""
+  pullPolicy: ""
+  pullSecrets: []
+
+# service only expose a metrics endpoint for prometheus to scrape,
+# starboard-operator does not have a user interface.
+service:
+  type: ClusterIP
+  metricsPort: 80
+  annotations:
+    prometheus.io/scrape: "true"
+    prometheus.io/path: /metrics
+
+# envSecret represent a k8s Secret resource that will be referenced to mount
+# environment variables.
+envSecret:
+  # create specifies whether a k8s Secret should be created with a given
+  # stringData. If this is set to false, only the keys defined in stringData
+  # will be optionally mounted from the named secret, rather than all keys in
+  # the named k8s Secret.
+  create: true
+  # name specifies the name of the k8s Secret to reference. If not set, a name
+  # is generated using the fullname template. name is currently required to be
+  # hardcoded to starboard-operator as the starboard-operator binary makes a
+  # hardcoded reference to that name:
+  # https://github.com/aquasecurity/starboard/blob/ad6b9b99931d7524806f79b1329fdda82c75a715/pkg/operator/scanner.go#L26-L28
+  name: "starboard-operator"
+  # stringData specifies key value pairs to be added to the k8s Secret and
+  # mounted by the Pod as environment variables.
+  #
+  # NOTE: It is important that you specify targetNamespace instead of
+  #       OPERATOR_TARGET_NAMESPACE below for this Helm chart to function
+  #       properly.
+  stringData:
+    OPERATOR_LOG_DEV_MODE: "false"
+    OPERATOR_SCANNER_TRIVY_ENABLED: "true"
+    OPERATOR_SCANNER_TRIVY_VERSION: "0.12.0"    # https://hub.docker.com/r/aquasec/trivy/tags
+    OPERATOR_SCANNER_AQUA_CSP_ENABLED: "false"
+    OPERATOR_SCANNER_AQUA_CSP_VERSION: ""
+    OPERATOR_SCANNER_AQUA_CSP_HOST: ""
+    OPERATOR_SCANNER_AQUA_CSP_USER: ""
+    OPERATOR_SCANNER_AQUA_CSP_PASSWORD: ""
+
+rbac:
+  create: true
```

Comment:
What' would be a legit use case to set this value to `false`? When someone granted cluster-admin permission to all service accounts via `service:serviceaccounts` group?

Reply:
When the account that deploys on regular basis doesnt have permissions to setup clusterroles for example i think.

I'm uncertain id this is a practice that remains helpful for some people still though, mostly added it because i know people have requested these aspects in the past for other helm charts.

Reply:
Okey so let's keep it.